web/app/controllers/users_controller.rb

```diff
@@ -41,6 +41,11 @@ def attempt_login
     end
   end
 
+  def logout
+    session[:current_user_email] = nil
+    redirect_to profile_user_path
```

Comment:
We should also set `@_current_user` to nil (take a look at https://github.com/adhocteam/soapbox/blob/master/web/app/controllers/application_controller.rb#L17)

Reply:
Good catch!